### ISSUE TITLE
Missing breaks in proxy switch statement

### ISSUE THREAD

**Issue by Tim3Game** [EmortalMC/Oblivion#1]
Hey,

I'm not sure if you tested Oblivion with the Velocity proxy using the `modern` forwarding-mode, but it didn't worked. At least it works now when I added breaks in the proxy switch statement. I was thinking of making a pull request but I have a 1.18.2 version of it now, and it's only like two lines, so it's probably easier to just write a Issue.

https://github.com/EmortalMC/Oblivion/blob/7c75c46646ef493d777c1543339fd1452f6ef7e3/src/main/java/dev/emortal/oblivion/OblivionMain.java#L73-L78

```java
switch (proxy) {
    case "velocity":
        VelocityProxy.enable(proxySecret);
        break;
    case "bungee":
        BungeeCordProxy.enable();
        break;
}
```

**Comment by emortaldev** [comment]
Oops I must have forgotten to commit this.

**Comment by emortaldev** [comment]
Fixed in 139ce89
